spec/factories/claims.rb

```diff
@@ -39,6 +39,12 @@
       has_disbursements { 'no' }
     end
 
+    Claim.states.each_key do |state|
```

Comment:
This is a really neat little addition to the factory.

Reply:
Also reminds me to do the same for PA